gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1043,9 +1045,13 @@ void gnc_file_ofx_import (GtkWindow *parent)
     DEBUG("gnc_file_ofx_import(): Begin...\n");
 
     default_dir = gnc_get_default_directory(GNC_PREFS_GROUP);
+    gtk_file_filter_set_name (filter, "ofx/qfx files (*.ofx,*.qfx)");
```

Comment:
Shouldn't this string be marked translatable ?

And related to this, I wonder if it wouldn't be better to use a more elaborate file type description rather than using the shorthand twice. I'd propose

```
_("Open/Quicken Financial Exchange file (*.ofx, *.qfx)")
```

Reply:
Did you implement these two suggestions before merging? ^^
I think you're right about them.

Reply:
I didn't implement anything. I'm just back from a two-week absence. I'm still catching up. And while doing so these two suggestions occurred to me.
Feel free to create a followup PR with these.

Reply:
In retrospect it would be overkill to make a PR for this one-liner, so don't bother. I have it queued to be pushed after 3.9 release is out.